src/main/java/duke/DukeOperations.java

```diff
@@ -0,0 +1,92 @@
+package duke;
+
+import java.util.ArrayList;
+
+/**
+ * Class containing all the various Duke program methods.
+ */
+public class DukeOperations {
+
+
+
+    /**
+    * method that returns exit message.
+    *
+    */
+    public void exit() {
```

Comment:
Perhaps can leave all the interactive sentences into a Ui class?  🤔 This may make the code easier to read. For example, a ui.showWelcomeMessage() method.